portfolio.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+ <head>
+	 <link rel="stylesheet" type="text/css" href="normalize.css">
+	 <link rel="stylesheet" type="text/css" href="portfolio.css"  >
+	 <title> Portfolio Simona Ambrosini</title>
+ </head>
+ 
+	
+ <body><
+	 <nav>
+		 <button>Menu</button>
+		 <div class="links">
+	       <a href="navbar.asp" style="text-decoration:none">Home</a>
+	 	   <a href="about.asp" style="text-decoration:none">About</a>
+	 	   <a href="portfolio.asp" style="text-decoration:none">Portfolio</a>
+	 	   <a href="contact.asp" style="text-decoration:none">Contact me</a>
+	 	</div>
+		<div class="download">
+			<a href="|">Download CV</a>
+		</div>
+	 </nav>
+	 <header>
+		 <div class="intro">
+		 <h1>Simona Ambrosini</h1>
+		 <h2></h2>
+	 </div>
+	 <section id="about" class="alt-bg">
+	     <div class="container">
+	         <div class="text">
+	             <h2>About</h2>
+				 <p></p>
+			 </div>
+		 </div>
+	 </section>
```

Comment:
Some formatting changes needed here, try to keep opening and closing tags at the same indentation. The class "alt-bg" is a bit confusing, try to be as clear as possible when naming classes. Also you have used many `<div>`s, the class="text" one is probably not needed as the `<h2>` and `<p>` are elements prepared to take and customise text.